htdocs/install/mysql/tables/llx_stock_mouvement.sql

```diff
@@ -37,5 +37,6 @@ create table llx_stock_mouvement
   fk_origin       integer,
   origintype      varchar(32),
   model_pdf       varchar(255),
-  fk_projet       integer NOT NULL DEFAULT 0
+  fk_projet       integer NOT NULL DEFAULT 0,
+  fk_qcstatus     VARCHAR(16)
```

Comment:
A status of control quality must be store on the product_lot.

Reply:
The problem here is that parts of a same lot can have a different qc_status. You can have 200 broken pieces in a 500 pieces lot. If it doesn’t fit in the stock_movement table, we shall consider a new table.

Reply:
I agree with @eldy  :
"The quality control is not something on the stock movement level."
This step can be carried out during other actions which could be:

- Remove from stock
- Re-invoice the customer
- etc ...

As such it seems to me that the initial quantity for the lot is missing
For example these quantities can be limited <1000 part for the "marque NF"

I think that could be a very good idea : 
"we shall consider a new table."
@eldy 
Do we offer you something like
Creation of a Quality Control table : 
-Numero lot
-Statut
-Quantité (par statut/par lot)

Reply:
@altairis-noe If you have 2 different lot with 2 different status, it can't be the same lot. Because, by definition a lot is the "smallest group of products that share the same properties". In such a cas the lot A will become Lot A and Lot A-Bad so 2 different lot.

Reply:
I’ll have to ask @lmag for his opinion, here, because I don’t understand lots that way. Products are sold in different quantities sharing an identifying lot number. If we create a lot each time part of lot is damaged or unavailable (even temporarily), I fear we lose pretty quick efficient traceability of the initial lot.

Reply:
Anyhow, that qcstatus question in a tricky one, and we will not resolve it here, I take it out of the PR